README.md

```diff
@@ -22,21 +22,11 @@ Use the following command to install them (assuming your control host has EPEL e
 * Master Hostname
 * CA certificate for the master
 
-Assuming your control host is your cluster master, you can run this snippet to get all the variables:
+Download [make_env.sh](https://raw.githubusercontent.com/container-mgmt/cm-ops-flow/master/make_env.sh), and run:
 
-```
-export OPENSHIFT_CFME_ROUTE="$(oc get route --namespace='openshift-management' -o go-template --template='{{.spec.host}}' httpd 2> /dev/null)"
-export OPENSHIFT_HAWKULAR_ROUTE="$(oc get route --namespace='openshift-infra' -o go-template --template='{{.spec.host}}' hawkular-metrics 2> /dev/null)"
-export OPENSHIFT_PROMETHEUS_ALERTS_ROUTE="$(oc get route --namespace='openshift-metrics' -o go-template --template='{{.spec.host}}' alerts 2> /dev/null)"
-export OPENSHIFT_PROMETHEUS_METRICS_ROUTE="$(oc get route --namespace='openshift-metrics' -o go-template --template='{{.spec.host}}' prometheus 2> /dev/null)"
-export OPENSHIFT_MASTER_HOST="$(oc get nodes -o name |grep master |sed -e 's/nodes\///g')"
-export OPENSHIFT_MANAGEMENT_ADMIN_TOKEN="$(oc sa get-token -n management-infra management-admin)"
-export OPENSHIFT_CA_CRT="$(cat /etc/origin/master/ca.crt)"
-export OPENSHIFT_PROVIDER_NAME="OCP37"
-export OPENSHIFT_CFME_USER="admin"
-export OPENSHIFT_CFME_PASS="smartvm"
-export OPENSHIFT_CFME_AUTH="${OPENSHIFT_CFME_USER}:${OPENSHIFT_CFME_PASS}"
-```
+`./make_env.sh | tee env_vars.sh` on your cluster master,
+
+and then `source env_vars.sh` on your control host (which can be the cluster master).
```

Comment:
currently this has to be the master, due to the cert command

Reply:
```
$ wget https://raw.githubusercontent.com/container-mgmt/cm-ops-flow/master/make_env.sh
$ bash make_env.sh
```

Reply:
make_env has to be run on the cluster master, but you can `source` the generated file on a different control host and it'd work too.